docs/cli/stacks/index.md

```diff
@@ -1,207 +1,132 @@
-
-Additionally, it enhances infrastructure management across multiple stacks and facilitates testing and deploying changes in a controlled manner, minimizing unintended consequences.
+# About Stacks
 
-The Terramate stack is a powerful feature that allows you to manage complex deployments.
+A modular approach is recommended when working with Infrastructure as Code (IaC). This approach breaks the entire infrastructure code and state into **smaller** and **isolated** units, often referred to as *stacks.*
 
+## What are Stacks?
 
-## What is a Stack?
+A stack is a collection of infrastructure resources that you *configure, provision* and *manage* as a unit.
 
-A stack is a collection of related resources managed as a single unit. When defining a stack, you specify all included resources, such as networks, virtual machines, and storage. 
-Stacks simplify resource creation and management, allowing you to focus on building and deploying the application.
+You can think about a stack as a combination of:
 
+- **Infrastructure code** which declares a set of infrastructure assets and their configuration.
+Terraform code (`.tf` files) and Cloud Formation (`.json` files) templates are examples of infrastructure code.
+- **State** that describes the status of the assets according to the *latest deployment* (e.g., Terraform state,
+Pulumi state, etc. - can be either stored locally or in a remote location)
+- **Configuration** to *configure* the infrastructure assets and stack behavior (e.g., Variables, Stack Configuration, etc.)
 
-## A Terramate stack is:
+Using stacks to break up your infrastructure code into manageable pieces is considered an industry standard and
+provides the following benefits:
 
-- A directory inside your project
-- Contains one or more Terramate configuration files
-- Includes a configuration file with a stack{} block
+**✅ Reduce run times significantly** by selectively targeting only the required stacks for execution (e.g. only the
+stacks that have changed in the last PR). Stacks also enable the possibility of parallel execution.
 
-The `stack{}` block distinguishes a stack from other directories in Terramate. By default, it doesn't require any attributes but can be used to describe the stack and orchestrate its execution.
+✅ **Limit the blast radius risk** by grouping IaC-managed assets in logical units such as environments, business units,
+regions or services that are isolated from each other.
 
-Stack configurations related to orchestration can be found [here](../orchestration/index.md). 
+✅ **Separate management responsibilities across team boundaries** by assigning and managing the ownership of stacks to
+users and teams.
 
+✅ **Remove sequential and blocking operations** by enabling parallel development and execution of independent stacks.
 
-## Why use Stacks?
+This page provides an overview of what Terramate Stacks are and how they help you create and manage composable
+Infrastructure as Code projects at any scale.
```

Comment:
This line seems redundant. As a reader I already expected this page to talk about the Terramate Stacks functionality, and the next heading makes it clear.

Reply:
I am suggesting to remove the line.

Reply:
makes sense - will remove